toshi_hazard_store/model/__init__.py

```diff
@@ -46,45 +47,45 @@ def drop_tables():
     drop_disagg()
 
 
-def configure_adapter(adapter_model: Type[PynamodbAdapterInterface]):
-    log.info(f"Configure adapter: {adapter_model}")
-    ensure_class_bases_begin_with(
-        namespace=openquake_models.__dict__,
-        class_name=str('ToshiOpenquakeMeta'),  # `str` type differs on Python 2 vs. 3.
-        base_class=adapter_model,
-    )
-    ensure_class_bases_begin_with(
-        namespace=location_indexed_model.__dict__, class_name=str('LocationIndexedModel'), base_class=adapter_model
-    )
-    ensure_class_bases_begin_with(
-        namespace=openquake_models.__dict__,
-        class_name=str('OpenquakeRealization'),  # `str` type differs on Python 2 vs. 3.
-        base_class=adapter_model,
-    )
-    ensure_class_bases_begin_with(
-        namespace=openquake_models.__dict__,
-        class_name=str('HazardAggregation'),
-        base_class=adapter_model,
-    )
+# def configure_adapter(adapter_model: Type[PynamodbAdapterInterface]):
```

Comment:
do you have a reason to keep all the commented out code?

Reply:
@chrisdicaprio  I'll do one more sweep I though I got rid of most of it